docs/using-the-compiler.rst

```diff
@@ -236,12 +236,12 @@ Input Description
             // `--allow-paths <path>`.
           ]
         },
-        "destructible":
+        "incrementable":
         {
           // Optional: keccak256 hash of the source file
           "keccak256": "0x234...",
           // Required (unless "urls" is used): literal contents of the source file
-          "content": "contract destructible is owned { function shutdown() { if (msg.sender == owner) selfdestruct(owner); } }"
+          "content": "contract incrementable is owned { uint256 private x = 0; function increment(uint256 _x) public { if (msg.sender == owner) x = _x; } }"
```

Comment:
Nothing truly wrong here except that from the name I expected to see the function to `++_x` and was confused for a moment why it does not do that :) Maybe this would be better:

```suggestion
          "content": "contract settable is owned { uint256 private x = 0; function set(uint256 _x) public { if (msg.sender == owner) x = _x; } }"
```

Same in `docs/metadata.rst`.

Reply:
ops :0